gui/wxpython/mapdisp/frame.py

```diff
@@ -111,23 +113,6 @@ def __init__(
         # Layer Manager layer tree object
         # used for VDigit toolbar and window and GLWindow
         self.tree = tree
-        # checks for saving workspace
-        self.canCloseDisplayCallback = None
-
-        # distinquishes whether map panel is dockable (Single-Window)
-        self._dockable = dockable
-
-        # distinguishes whether map panel is docked or not
-        self._docked = True
-
-        # undock/dock bound method
-        self._docking_callback = None
-
-        # Saved Map Display output img size
-        self._saved_output_img_size = None
```

Comment:
It looks like this disappeared and it's causing an error when saving image in map display:
```
Traceback (most recent call last):
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-
gnu/gui/wxpython/mapdisp/frame.py", line 729, in SaveToFile

dlg = ImageSizeDialog(self,
img_size=self._saved_output_img_size)
AttributeError
:
'MapPanel' object has no attribute '_saved_output_img_size'
```

Reply:
Thanks for testing. Fixed in 78ad9a7be2.